domain_models/fields.py

```diff
@@ -160,11 +164,18 @@ def __init__(self, related_model_cls, default=None, required=False):
         self.related_model_cls = related_model_cls
 
     def _converter(self, value):
-        """Convert raw input value of the field."""
+        """Convert raw input value of the field.
+
+        :param object value:
+        :rtype object:
+        """
+        value = self.related_model_cls(**value) if isinstance(value,
```

Comment:
Hey @boonya,

You know, man, I got some feeling that it is too long and this ternary operator doesn't suite this situation.

Let's try to rework it with standard ``if`` statement. Hope it will become more readable.